src/components/Editor/ClientCursor.ts

```diff
@@ -29,57 +31,69 @@ class ClientCursor {
     this.removeCursor();
     const cursorCoords = cm.cursorCoords(cursorPos);
     const cursorElement = document.createElement('span');
-    const size = cursorCoords.bottom - cursorCoords.top;
+    this.height = cursorCoords.bottom - cursorCoords.top;
 
     cursorElement.style.position = 'absolute';
     cursorElement.style.borderLeftStyle = 'solid';
-    cursorElement.style.borderLeftWidth = '2px';
+    cursorElement.style.borderLeftWidth = '2.5px';
     cursorElement.style.borderLeftColor = this.color;
-    cursorElement.style.height = `${size}px`;
+    cursorElement.style.height = `${this.height}px`;
     cursorElement.style.padding = '0px';
-    cursorElement.style.zIndex = '0';
 
-    const nameBoxElement = document.createElement('span');
-    nameBoxElement.style.position = 'absolute';
-    nameBoxElement.style.top = `-${size}px`;
-
-    const nameElement = document.createElement('span');
-    nameElement.textContent = this.id;
-    nameElement.style.position = 'fixed';
-    nameElement.style.backgroundColor = this.color;
-    nameElement.style.padding = '1px 4px';
-    nameElement.style.borderRadius = '4px';
-    nameElement.style.color = invert(this.color, true);
-    nameElement.style.animationDuration = `${duration}s`;
-    nameElement.style.animationDelay = `${delay}s`;
-    nameElement.className = 'text-remove';
-
-    nameBoxElement.appendChild(nameElement);
-    cursorElement.appendChild(nameBoxElement);
-
-    this.removeNameReserve(nameBoxElement);
+    this.showCursorNameReserve(cursorElement);
     this.marker = cm.setBookmark(cursorPos, {
       widget: cursorElement,
       insertLeft: true,
     });
   }
 
-  updateLine(cm: CodeMirror.Editor, fromPos: CodeMirror.Position, toPos: CodeMirror.Position) {
+  updateLine(
+    cm: CodeMirror.Editor,
+    fromPos: CodeMirror.Position,
+    toPos: CodeMirror.Position,
+  ) {
     this.removeLine();
     this.lineMarker = cm.getDoc().markText(fromPos, toPos, {
       css: `background-color : ${this.color}; opacity:0.7`,
     });
   }
 
+  // when user's cursor hover, show name
+  showCursorNameReserve(parentEl: Element) {
+    const nameElement = document.createElement('span');
+
+    parentEl.addEventListener('mouseenter', () => {
+      nameElement.textContent = this.id;
+      nameElement.style.position = 'absolute';
+      nameElement.style.top = `-${this.height}px`;
+      nameElement.style.left = `-2px`;
```

Comment:
Same as the comment below.
https://github.com/yorkie-team/yorkie-codepair/pull/17#discussion_r532165588